pkg/cmd/server/api/types.go

```diff
@@ -1202,3 +1202,13 @@ type ServiceServingCert struct {
 	// If this value is nil, then certs are not signed automatically.
 	Signer *CertInfo
 }
+
+// AdmissionPluginActivation can be used to enable to disable various admission plugins.
+// When this type is present as the `configuration` object under `pluginConfig` and *if* the admission plugin supports it,
+// this will cause an off by default admission plugin to be enabled
+type AdmissionPluginActivation struct {
```

Comment:
On first glance the name doesn't seem to communicate what its effect is.  It's going to be the kind inside of a file?

I don't like that it's called pluginConfig (for starters).  That really should have been AdmissionConfig or similar.  I don't want to propagate plugin wording beyond that.  This is really an empty default config file for an admission controller, but I don't see a mix of those words that is approachable.

DefaultAdmissionConfig is probably as close as I can get to what this is.  Counter arguments?